VeteransCenters/veterans_centers.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+import logging
+from io import StringIO
+
+logger = logging.getLogger(__name__)
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, get_mongo_client
+)
+from shared_code.base_scraper import BaseScraper
+
+api_key = os.environ['VETERANS_API_KEY']
+
+class VeteransCentersScraper(BaseScraper):
+
+    def grab_data(self):
+        url = 'https://sandbox-api.va.gov/services/va_facilities/v0/facilities/all'
```

Comment:
We need production access.

Reply:
Raj said we would get the production access once later on, that is why for the moment I did it this way, to check it works, and in the future update the url tot he production one.

Reply:
In that case I would change the name of the tmp tables to indicate testData may be something like `tmpVeteransSandbox` or `tmpVeteransTestData` etc so that we don't mix it with production.

Reply:
done!